pkg/util/template/template.go

```diff
@@ -0,0 +1,12 @@
+package template
+
+import (
+	"github.com/maistra/maistra-test-tool/pkg/util"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func Run(t test.TestHelper, yaml string, vars interface{}) string {
+	t.T().Helper()
+	template := util.RunTemplate(yaml, vars)
+	return template
```

Comment:
no need for variable, just return directly

Reply:
Done